applications/nrf_desktop/bootloader_dfu.rst

```diff
@@ -203,6 +203,9 @@ You can enhance security further by enabling the following sysbuild Kconfig opti
   To use KMU, the public key must first be provisioned.
   See the :ref:`ug_nrf54l_developing_provision_kmu` documentation for details.
 
+  .. note::
+     The KMU feature is only supported for the nRF54L SoC Series.
```

Comment:
Consider combining the note with the bullet point above (to keep the information together)